tutorials/navigation/navigation_using_navigationpaths.rst

```diff
@@ -0,0 +1,125 @@
+.. _doc_navigation_using_navigationpaths:
+
+Using NavigationPaths
+=====================
+
+Obtaining a Navigationpath
+--------------------------
+
+Navigation paths can be directly queried from the NavigationServer do not require any 
+additional nodes or objects as long as the navigation map as a navigationmesh to work with.
+
+To obtain a 2D path use ``NavigationServer2D.map_get_path(map, from, to, optimize, navigation_layers)``.
+
+To obtain a 3D path use ``NavigationServer3D.map_get_path(map, from, to, optimize, navigation_layers)``.
+
+One of the required parameters for the query is the RID of the navigation map. 
+Each game ``World`` has a default navigation map automatically created. 
+The default navigation maps can be received with ``get_world_2d().get_navigation_map()`` from 
+any Node2D inheriting node or ``get_world_3d().get_navigation_map()`` from any Node3D inheriting node.
+The second and third parameters are the starting position and the target position as Vector2 for 2D or Vector3 for 3D.
+
+
+If ``optimized`` parameter is ``true`` path positions will be shortened along polygon 
+corners with an additional funnel algorithm pass. This works well for free movement 
+on navigationmeshes with unequal sized polygons as the path will hug around corners 
+along the polygon corridor found by the A* algorithm. With small cells the A* algorithm 
+creates a very narrow funnel corridor that can create ugly corner paths when used with grids.
+
+If ``optimized`` parameter is ``false`` path positions will be placed at the center of each passed polygon edge. 
```

Comment:
This part isn't clear to me. does "center of each passed polygon edge" just mean the center of navigation grid cells?

Reply:
Godot currently adds a path vector point every time the pathfinding crosses from one polygon edge to another polygon. By default it will use the shortest distance to the edge but when optimize=false it will move the point to the middle of the entire edge, good for grid cells, horrible for large open world polygons.